src/main/java/org/broadinstitute/consent/http/db/mapper/DarCollectionSummaryReducer.java

```diff
@@ -44,6 +49,28 @@ public void accumulate(Map<Integer, DarCollectionSummary> map, RowView rowView)
           summary.addReferenceId(darReferenceId);
         }
 
+        if (hasColumn(rowView, "dar_all_reference_id", String.class)) {
+          aggReferenceId = rowView.getColumn("dar_all_reference_id", String.class);
+          if (Objects.nonNull(aggReferenceId)) {
+            summary.addReferenceId(aggReferenceId);
+            if (hasColumn(rowView, "dar_all_parent_id", String.class)) {
+              aggParentId = rowView.getColumn("dar_all_parent_id", String.class);
+              if (Objects.nonNull(aggParentId)) {
+                summary.addParentToReferenceId(aggParentId, aggReferenceId);
+              } else {
+                summary.addParentToReferenceId(null, aggReferenceId);
```

Comment:
I should probably switch the key to this entry to the id of the DAR for now, updating it to the referenceId when we have it.
From a graph theory perspective, switching to the referenceId for keys makes this a much cleaner implementation.  That will fall in naturally when we move parent_id to hold the referenceId.  I don't think the alternative of using the data_access_request.id field makes sense for the Set values because the front-end code and our APIs already have access patterns based on the referenceId

Reply:
Reverted building out this implementation based on conversation in ticket.